prime-router/settings/STLTs/AR/AR-phd.yml

```diff
@@ -0,0 +1,66 @@
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "ar-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "azure:hl7_mapping/receivers/STLTs/AR/AR-receiver-transform.yml"
+        format: "hl7"
+        useBatchHeaders: false
+        defaults: { }
+        nameFormat: "STANDARD"
+        convertDateTimesToReceiverLocalTime: true
+        receivingOrganization: null
+        type: HL7
+        truncationConfig:
+          truncateHDNamespaceIds: true
+          truncateHl7Fields: [ "ORC-3-2, ORC-21-1, OBR-3-2, OBX-15-2, OBX-17-1, OBX-23-1" ]
+          customLengthHl7Fields: { }
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(Patient).address.state.exists() and Bundle.entry.resource.ofType(Patient).address.state = 'AR') or (Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state.exists() and Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'AR')"
+      qualityFilter:
+        - "Bundle.entry.resource.ofType(DiagnosticReport).result.resolve().where(method.empty() or value.coding.code.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(DiagnosticReport).where( basedOn.resolve().extension('https://reportstream.cdc.gov/fhir/StructureDefinition/effective-date').empty() and specimen.resolve().collection.collected.empty() and effective.empty() and issued.empty() and result.resolve().effective.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(Patient).where(address.line[0].empty() and address.postalCode.empty() and telecom.where(system = 'phone').value.empty() and telecom.where(system = 'email').value.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(Patient).where(name.family.empty() or name.given.empty() or birthDate.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(ServiceRequest).performer.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA'"
+        - "Bundle.entry.resource.ofType(ServiceRequest).requester.resolve().organization.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA'"
+        - "Bundle.entry.resource.ofType(ServiceRequest).where(requester.resolve().practitioner.resolve().name.family.empty()).count()= 0"
+        - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+        - "Bundle.identifier.value.empty().not()"
+      routingFilter:
+        - "Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01'"
+      processingModeFilter: [ ]
+      conditionFilter:
+        #send all observations if there exists at least one observation where condition code = 6142004 (Influenza), 541131000124102 (Influenza A), 840539006 (COVID-19, excluding antibody tests), 414015000 (Orthopox Viruses) or 86406008 (HIV) AND result is abnormal (POSITIVE or DETECTED).
```

Comment:
First time I see this included 414015000 (Orthopox Viruses). What is that for, when do we need to include it?